server.go

```diff
@@ -3355,7 +3359,7 @@ func (s *Server) DiagnosticsAsRows() []*influxql.Row {
 			nodes = append(nodes, strconv.FormatUint(n, 10))
 		}
 		var path string
-		if sh.store != nil {
```

Comment:
Not strictly necessary, but standardizes on using `HasDataNodeID` to check if a shard is local.